--- a/data/com.frac_tion.teleport.appdata.xml.in	2020-01-15 17:35:27.259771155 +0100
+++ b/data/com.frac_tion.teleport.appdata.xml.in	2020-01-15 17:36:14.469704365 +0100
@@ -36,7 +36,7 @@
     <kudo>ModernToolkit</kudo>
     <kudo>Notifications</kudo>
   </kudos>
-
+  <content_rating type="oars-1.0" />
   <update_contact>julian@sparber.net</update_contact>
   <translation type="gettext">teleport</translation>
 
